nncf/common/graph/graph.py

```diff
@@ -170,10 +176,11 @@ def traverse_graph(self,
                        traverse_function: Callable[[NNCFNode, List[Any]], Tuple[bool, List[Any]]],
                        traverse_forward: bool = True):
         """
-        Traverses graph up or down starting form `curr_node` node
-        :param curr_node: node from which traverse is started
-        :param traverse_function: function describing condition of traverse continuation/termination
-        :param traverse_forward: flag specifying direction of traverse
+        Traverses graph up or down starting form `curr_node` node.
+
+        :param curr_node: Node from which traverse is started.
+        :param traverse_function: Function describing condition of traverse continuation/termination.
+        :param traverse_forward: Flag specifying direction of traverse.
```

Comment:
traversal in each of the three lines above

Reply:
Thanks! I fixed it.